modes/consult/evil-collection-consult.el

```diff
@@ -36,6 +36,45 @@
   (evil-set-command-property 'consult-mark :jump t)
   (evil-set-command-property 'consult-line :jump t))
 
+(defun evil-mark-ring ()
```

Comment:
This functionality seems pretty cool, I think we'd need to prefix these with evil-collection-consult though as to not pollute the global namespace which begs the question of where this functionality belongs.

Reply:
Right, I don't think the maintainers of consult want to include any evil configuration in their repo, which makes sense. So I figured this would be the best place to put it. However, I understand if you want to restrict it to just keybindings.

Reply:
Right, I was just thinking the usefulness of this function is partly also thanks to a short name, once you add the -collection- part, it becomes quite verbose as an interactive function. 

I'm guessing it's not a problem for you since you probably have the function bound somewhere else. If the maintainers don't want this, this should be an OK spot, I think.